experimental/Schemes/elliptic_surface.jl

```diff
@@ -1599,22 +1592,6 @@ on the curve defined by `g`, i.e. `g(P) == 0`.
 function transform_to_weierstrass(g::MPolyRingElem, x::MPolyRingElem, y::MPolyRingElem, P::Vector{<:RingElem})
   R = parent(g)
   F = fraction_field(R)
-
-  # In case of variables in the wrong order, switch and transform the result.
-  if x == R[2] && y == R[1]
-    switch = hom(R, R, reverse(gens(R)))
-    g_trans, trans = transform_to_weierstrass(switch(g), y, x, reverse(P))
-    new_trans = MapFromFunc(F, F, f->begin
-                                switch_num = switch(numerator(f))
-                                switch_den = switch(denominator(f))
-                                interm_res = trans(F(switch_num))//trans(F(switch(den)))
-                                num = numerator(interm_res)
-                                den = denominator(interm_res)
-                                switch(num)//switch(den)
-                            end
-                           )
-    return switch(g_trans), new_trans
-  end
   @assert ngens(R) == 2 "input polynomial must be bivariate"
```

Comment:
Where did this code go? How was the issue resolved that lead to introducing it back then?

Reply:
it was duplicated below